engine/table/python/core/deephaven_jpy_init.py

```diff
@@ -18,188 +16,5 @@
 # If you want jpy to tell you about all that it is doing, change this
 # jpy.diag.flags = jpy.diag.F_ALL
 
-from deephaven import Config
-import deephaven.TableTools as ttools
-from deephaven.python_to_java import dataFrameToTable
-
-# NOTE: **THIS IS REQUIRED** for WorkerPythonEnvironment - don't take it out...
-import __main__
-
-
-###############################################################################################
-
-# can't use IsWidget.get_dh_table or the data frame will be interpreted as a table instead of a widget?
-pandas.DataFrame.to_dh_table = lambda self: dataFrameToTable(self, True)
-
-
-########################################################################################################
-# Performance monitoring
-
-def add_tables_from_map_to_binding(tables):
-    """
-    Iterates through a (java) map of tables and adds them to the global binding. This is a helper method
-    accommodating for the fact that the performance queries return the table results as a (java) HashMap object.
-
-    :param tables: java HashMap
-    """
-
-    it = tables.entrySet().iterator()
-    while it.hasNext():
-        pair = it.next()
-        globals()[pair.getKey()] = pair.getValue()
-        it.remove()
-
-
-def query_update_performance_set(evaluationNumber):
-    """
-    Name matched to Groovy equivalent for coherent usage.
-    TODO: document this
-
-    :param worker_name:
-    :param date:
-    :param use_intraday:
-    :param server_host:
-    :param as_of_time:
-    :return:
-    """
-
-    _jtype_ = jpy.get_type("io.deephaven.engine.table.impl.util.PerformanceQueries")
-
-    tableMap = _jtype_.queryUpdatePerformanceMap(evaluationNumber)
-    add_tables_from_map_to_binding(tableMap)
-
-# Convenience functions for a variety of purposes
-
-
-def importjava(clazz):
-    _jclass_ = jpy.get_type("java.lang.Class")
-    ql = jpy.get_type("io.deephaven.engine.table.lang.QueryLibrary")
-
-    def _get_short_class_name_(clazz):
-        return _jclass_.forName(clazz).getSimpleName()
-
-    clazz = clazz.strip()
-    javaclass = _get_short_class_name_(clazz)
-    globals()[javaclass] = jpy.get_type(clazz)
-    ql.importClass(_jclass_.forName(clazz))
-
-
-def importstatic(clazz):
-    _jclass_ = jpy.get_type("java.lang.Class")
-    ql = jpy.get_type("io.deephaven.engine.table.lang.QueryLibrary")
-    clazz = clazz.strip()
-    javaclass = jpy.get_type(clazz)
-    for key, value in javaclass.__dict__.items():
-        if not key.startswith("__"):
-            if hasattr(value, 'methods'):
-                methods = value.methods
-                if methods:
-                    if methods[0].is_static:
-                        globals()[value.name] = value
-
-    ql.importClass(_jclass_.forName(clazz))
```

Comment:
`QueryLibrary` allows java functions to be added to the query language.  We might want to wrap this to allow more powerful multi-language integrations.  It should be in its own module and not here.  Create a ticket for it.